packages/material-ui/src/FormControl/FormControl.js

```diff
@@ -229,6 +231,10 @@ FormControl.propTypes = {
    * If `true`, the label should be displayed in an error state.
    */
   error: PropTypes.bool,
+  /**
+   * If `true`, the component will be displayed in focused state.
+   */
+  forceFocus: PropTypes.bool,
```

Comment:
I wonder if `focused` would be enough.

Reply:
Yeah I think `focused` will not be enough and can confuse somebody, I even thought about `forceFocusedStyle` or `forceFocusedAppearance`